metadata-db-app/src/cmr/metadata_db/api/concepts.clj

```diff
@@ -95,7 +95,10 @@
 (defn- force-delete
   "Permanently remove a concept version from the database."
   [context params concept-id revision-id]
-  (let [{:keys [revision-id]} (concept-service/force-delete context concept-id (as-int revision-id))]
+  (let [;; force? is only used for testing purpose, should not be used in real operations
```

Comment:
I think this is the most expedient way to do this right now ... but maybe in the future we can remove support for this and add a utility function in the tests that allows us to make direct, custom queries to the DB in order to induce inconsistencies against which we can test ...